frontend/components/commons/forms/questions-form/QuestionsForm.component.vue

```diff
@@ -251,6 +268,32 @@ export default {
         default:
       }
     },
+    async onClear() {
+      const responseId = getRecordResponsesIdByRecordId({
+        userId: this.userId,
+        recordId: this.recordId,
+      });
+
+      try {
+        const responseData =
+          responseId && (await this.deleteResponsesByResponseId(responseId));
+
+        await deleteRecordResponsesByUserIdAndResponseId(
+          this.userId,
+          responseData?.data?.id
+        );
+
+        // NOTE - onClear event => the status change to pending
+        await updateRecordStatusByRecordId(
```

Comment:
It would be possible add this also to the other form actions? (change to submitted)